.expeditor/end_to_end.pipeline.yml

```diff
@@ -1,25 +1,122 @@
 expeditor:
+  secrets:
+    # TODO: change this once we go to production
+    ACCEPTANCE_HAB_AUTH_TOKEN:
+      path: account/static/habitat/chef-ci
+      field: scotthain-sig-key
   defaults:
     buildkite:
       timeout_in_minutes: 30
+      env:
+        HAB_ORIGIN: "core"
+        ACCEPTANCE_HAB_BLDR_URL: "https://bldr.acceptance.habitat.sh"
+        CI_OVERRIDE_CHANNEL: "DEV"
+        HAB_BLDR_CHANNEL: "DEV"
+        HAB_INTERNAL_BLDR_CHANNEL: "DEV"
 
 steps:
 #######################################################################
-# E2E!
+# E2E
 #######################################################################
 
-  - label: "[First step]"
+  - label: "[:linux: test_hab_help_doesnt_install_hab_sup]"
     command:
-      - echo "HI"
-    agents:
-      queue: 'default-privileged'
-    plugins:
-      docker#v3.0.1:
-        always-pull: true
-        user: "buildkite-agent"
-        group: "buildkite-agent"
-        image: "ubuntu"
-    timeout_in_minutes: 10
-    retry:
-      automatic:
-        limit: 1
\ No newline at end of file
+      - .expeditor/scripts/setup_environment.sh DEV
+      - test/end-to-end/test_hab_help_doesnt_install_hab_sup.sh /bin/hab
+    expeditor:
+      executor:
+        docker:
+          privileged: true
+          propagate-environment: true
```

Comment:
`propagate-environment` is `true` by default in our setup (managed by Expeditor)